.vale/styles/OpenShiftAsciiDoc/MergeConflictMarkers.yml

```diff
@@ -0,0 +1,8 @@
+---
+extends: existence
+scope: raw
+level: error
+link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line
+message: "Remove merge conflict markers from documentation"
+tokens:
+  - '<<<<<<< HEAD, >>>>>>>'
```

Comment:
https://studio.vale.sh/s/c9ee4e5ab886bff43606a8a127e6c1b6 - see this, you can have multiple tokens. 
Start of line regex ^
End of line regex $
Exact number of {7}

So for example, `^<{7}$` would match exactly 7 `<` from start of a line to end of a line